modules/web/package.json

```diff
@@ -44,14 +44,14 @@
   },
   "dependencies": {
     "@angular/animations": "16.2.1",
-    "@angular/cdk": "16.2.1",
+    "@angular/cdk": "~16.2.1",
```

Comment:
Could you specify version strictly? Please remove `~`.

Reply:
Hi, I've made all the needed changes here https://github.com/kubernetes/dashboard/pull/8345